### PR TITLE
Fix stakeholder preview getting cut off

### LINKED ISSUE #2093: Content Cut Off on Stakeholder Details Mobile View
### Overview
When viewing the Stakeholder Details on Mobile, the bottom of the content gets cut off. This likely because the bottom of the div is off screen.

### Action Items
- [ ] Make sure all the stakeholder details are viewable on mobile; the bottom of the drawer should be the bottom of the view portal.
- [ ] This should work regardless of the size of the drawer

### Resources/Instructions
What it looks like on Mobile:
chrome_qEXK4EMcy6

What it looks like on Desktop without info cut off:
chrome_XCzZYkDyCs

### PR DESCRIPTION
**Pull request by hanapotski**
Closes #2093

### CODE REVIEW

**Review by andyarensman — CHANGES_REQUESTED** [review]
Double scroll bar is now showing

**Review by andyarensman — CHANGES_REQUESTED** [review]
The double scroll bar is gone, but the information is still being cut off for me:

chrome_SCgmHTvgTw

**Comment by hanapotski** [comment]
🤦‍♀️ will look into it more 🧐 I think we need to dynamically change the height of the list container

**Comment by 93Belen** [comment]
I added the position of the Draggable component to the App State, and then use it to dynamically change the padding of the Stake Holder Details. If you guys test it in the Desktop with the screen as Mobile, the padding looks like is too much, but it works on the actual phone device. At least in mine :)
Also, I still have double scrolling in the phone, no in the Desktop as phone.

https://github.com/hackforla/food-oasis/assets/98275292/df758efc-07e9-4f71-aa99-d51949afea6c

File: client/src/components/FoodSeeker/SearchResults/StakeholderDetails/StakeholderDetails.js [hunk 1]
```diff
@@ -88,6 +89,37 @@ const StakeholderDetails = ({ onBackClick, isDesktop }) => {
...
+  //     newY = 25;
+  //   } else if (ui.y > 40 * windowHeight) {
+  //     newY = 57;
+  //   }
+  //   setPosition({ x: 0, y: newY * windowHeight });
+  // };
+
+
+
+  console.log(position)
```
Comment (andyarensman) [inline client/src/components/FoodSeeker/SearchResults/StakeholderDetails/StakeholderDetails.js:122]: Leftover console.log

**Comment by 93Belen** [comment]
@andyarensman I removed it, thank you! :)

**Review by andyarensman — APPROVED** [review]
LGTM. I  guess we can fix the phone double scroll in a separate issue? @hanapotski

**Pull request merged**